CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 3.0.0
+- New experimental, pure Kotlin `mvrx-core` module (#635)
```

Comment:
I think we should call out what it's for (MavericksRepository)
Could you also add a line for Mario's `argsOrNull()` pr? https://github.com/airbnb/mavericks/pull/639